{{cookiecutter.project_name}}/Data/sample_treatment/simple_repeat.py

```diff
@@ -30,8 +20,24 @@ def silent_cmd(command):
     R script calls unitl all neccesary data is created.'''
     return subprocess.Popen(command, shell=True, stdout=subprocess.PIPE).wait()
 
+start_range = 10
+end_range = 21
+
+if(len(sys.argv) > 1):
+    start_range = int(sys.argv[1])
+    if(len(sys.argv) > 2):
+        end_range = int(sys.argv[2]) + 1
```

Comment:
probably good to include a comment as to why this is happening

Reply:
Comment included!